Bottom: fcc1405c5d9f2f6be84f38c7681cd4b2d65efd1e
Top:    d9b5f706fc9446bb34e94e9f3a9ade932485f7c2
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:48:09 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index 111ea4f7..2a157e96 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -1,3 +1,5 @@
+buildbot-worker.novatech-llc.com ansible_host=127.0.0.1 ansible_port=20022 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/bbworker/virtualbox/private_key'
+
 [bacula]
 testdaedalus.novatech-llc.com
